monai/inferers/inferer.py

```diff
@@ -49,13 +49,13 @@ class SimpleInferer(Inferer):
     def __init__(self) -> None:
         Inferer.__init__(self)
 
-    def __call__(self, inputs: torch.Tensor, network: torch.nn.Module):
+    def __call__(self, inputs: torch.Tensor, network: Callable[[torch.Tensor], torch.Tensor]):
```

Comment:
I am not sure whether `Callable[[torch.Tensor], torch.Tensor]` is a suitable typehint.
Maybe `Callable[Sequence[torch.Tensor], torch.Tensor]` is more clear?